src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-revisions-controller.php

```diff
@@ -419,133 +342,36 @@ public function get_item_schema() {
 			return $this->add_additional_fields_schema( $this->schema );
 		}
 
-		$schema = array(
-			'$schema'    => 'http://json-schema.org/draft-04/schema#',
-			'title'      => "{$this->parent_post_type}-revision",
-			'type'       => 'object',
-			// Base properties for every revision.
-			'properties' => array(
-
-				/*
-				 * Adds settings and styles from the WP_REST_Revisions_Controller item fields.
-				 * Leaves out GUID as global styles shouldn't be accessible via URL.
-				 */
-				'author'       => array(
-					'description' => __( 'The ID for the author of the revision.' ),
-					'type'        => 'integer',
-					'context'     => array( 'view', 'edit', 'embed' ),
-				),
-				'date'         => array(
-					'description' => __( "The date the revision was published, in the site's timezone." ),
-					'type'        => 'string',
-					'format'      => 'date-time',
-					'context'     => array( 'view', 'edit', 'embed' ),
-				),
-				'date_gmt'     => array(
-					'description' => __( 'The date the revision was published, as GMT.' ),
-					'type'        => 'string',
-					'format'      => 'date-time',
-					'context'     => array( 'view', 'edit' ),
-				),
-				'id'           => array(
-					'description' => __( 'Unique identifier for the revision.' ),
-					'type'        => 'integer',
-					'context'     => array( 'view', 'edit', 'embed' ),
-				),
-				'modified'     => array(
-					'description' => __( "The date the revision was last modified, in the site's timezone." ),
-					'type'        => 'string',
-					'format'      => 'date-time',
-					'context'     => array( 'view', 'edit' ),
-				),
-				'modified_gmt' => array(
-					'description' => __( 'The date the revision was last modified, as GMT.' ),
-					'type'        => 'string',
-					'format'      => 'date-time',
-					'context'     => array( 'view', 'edit' ),
-				),
-				'parent'       => array(
-					'description' => __( 'The ID for the parent of the revision.' ),
-					'type'        => 'integer',
-					'context'     => array( 'view', 'edit', 'embed' ),
-				),
+		$schema               = parent::get_item_schema();
+		$parent_schema        = $this->parent_controller->get_item_schema();
+		$schema['properties'] = array_merge( $schema['properties'], $parent_schema['properties'] );
 
-				// Adds settings and styles from the WP_REST_Global_Styles_Controller parent schema.
-				'styles'       => array(
-					'description' => __( 'Global styles.' ),
-					'type'        => array( 'object' ),
-					'context'     => array( 'view', 'edit' ),
-				),
-				'settings'     => array(
-					'description' => __( 'Global settings.' ),
-					'type'        => array( 'object' ),
-					'context'     => array( 'view', 'edit' ),
-				),
-			),
-		);
+		unset( $schema['properties']['guid'] );
+		unset( $schema['properties']['slug'] );
+		unset( $schema['properties']['meta'] );
+		unset( $schema['properties']['content'] );
+		unset( $schema['properties']['title'] );
```

Comment:
Nit: same here, can use one `unset()` call for all of these

Reply:
Also did this one over in https://github.com/WordPress/wordpress-develop/pull/6573. Thanks!

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/6105/commits/c62fed9b8f8c1403f7ee4c06151d833a9b8fcd3f